doc/source/index.rst

```diff
@@ -6,10 +6,23 @@
 Welcome to Language Model Playground's documentation!
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
+and it can generate sentences, evaluation models, inference words. 
```

Comment:
~evaluataion~ evaluate.
Remove `inference words` and add following utilites:

- Train tokenizer
- Train neural network based language model
- Generate text using language model
- And more to come